### DISCUSSION TITLE
Make vcpkg install system packages itself

### DISCUSSION THREAD

**Discussion by adorostkar** [microsoft/vcpkg#22152] [Q&A]
Hello all,
I am starting to use vcpkg and I wonder if there is an option to tell vcpkg to install system packages by itself.

I see when installing lets say libpoxy that the build fails because libglew is missing. Is there any way to let vcpkg know that I want all the missing libs to be installed? 

I know conan has this option which in my opinion makes it a bit more automatic when using.

**Comment by autoantwort** [comment]
No there is no such option
